pygmt/datasets/earth_age.py

```diff
@@ -54,9 +54,14 @@ def load_earth_age(resolution="01d", region=None, registration=None):
 
     Note
     ----
-    The :class:`xarray.DataArray` grid doesn't support slice operation, for
-    Earth seafloor crustal age with resolutions of 5 arc-minutes or higher,
-    which are stored as smaller tiles.
+
+    The registration and coordinate system type of the returned
+    :class:`xarray.DataArray` grids can be accessed via the GMT accessors
+    (i.e., ``grid.gmt.registration`` and ``grid.gmt.gtype`` respectively).
+    However, these properties may be lost after specific grid operations and
+    need to be set manually before passing the grid further to PyGMT functions.
+    Refer to :class:`pygmt.GMTDataArrayAccessor` for detailed explanations and
+    workarounds.
```

Comment:
Recommendation to still mention slicing as an example of an operation that causes `registration` and `gtype` to be lost, and be explicit that this affects **both** PyGMT [data processing](https://www.pygmt.org/v0.8.0/api/index.html#data-processing) and [plotting](https://www.pygmt.org/v0.8.0/api/index.html#plotting) functions.

```suggestion
    However, these properties may be lost after specific grid operations (such
    as slicing) and will need to be manually set before passing the grid to any
    PyGMT data processing or plotting functions. Refer to
    :class:`pygmt.GMTDataArrayAccessor` for detailed explanations and
    workarounds.
```